sema/bench/__main__.py

```
@@ -0,0 +1,108 @@
+
+def make_service(args: Namespace) -> Sembench:
+    return Sembench(
+        locations=args.locations,
+        sembench_config_file_name=args.config_name,
+        scheduler_interval_seconds=args.interval,
+        watch_config_file=args.watch,
+        fail_fast=args.fail_fast,
+    )
+
+
+def _main(*args_list) -> bool:
```

Ruff

Missing type annotation for `*args_list`

(ANN002)